parallel_test.go

```diff
@@ -0,0 +1,91 @@
+package testcontainers
+
+import (
+	"context"
+	"testing"
+)
+
+func TestParallelContainers(t *testing.T) {
+	tests := []struct {
+		name      string
+		reqs      ParallelContainerRequest
+		resLen    int
+		expErrors int
+	}{
+		{
+			name: "running two containers (success)",
+			reqs: ParallelContainerRequest{
+				{
+					ContainerRequest: ContainerRequest{
+
+						Image: "nginx",
+						ExposedPorts: []string{
+							"10080/tcp",
+						},
+					},
+					Started: true,
+				},
+				{
+					ContainerRequest: ContainerRequest{
+
+						Image: "nginx",
+						ExposedPorts: []string{
+							"10081/tcp",
+						},
+					},
+					Started: true,
+				},
+			},
+			resLen: 2,
```

Comment:
Although the default int value is zero, I'd add it here explicitly for readability:
```suggestion
			resLen: 2,
			expErrors: 0,
```